### ISSUE TITLE
Links to PDF versions should be labelled something more descriptive than "PDF"

### ISSUE THREAD

**Issue by christianp** [chirun-ncl/chirun#112]
The links to PDFs on the table of contents should be labelled with the name of the item they correspond to

**Comment by christianp** [comment]
I think this doesn't apply to the new theme.
